### ISSUE TITLE
[iOS] Send funds: enable users to choose a recipient from the address book and to add "ENS name" option

### ISSUE THREAD

**Issue by the-real-chrizzo** [safe-global/safe-ios#3015]
As a user
I want to be able to choose a recipient from my address book or choose "ENS name"
to send funds more easily

**Background**
When sending funds a user can only choose from "Paste from Clipboard" or "Scan QR Code" when clicking on the "Recipient's address" while sending funds. With this feature we want the user to be able to also choose from "Address Book" similar to the user flow when adding an owner or "ENS name" as recipient address.

**What needs to be implemented**
Ability to choose a contact form the address book as recipient of a transaction
Ability to choose an ENS name as recipient

**Acceptance criteria**
Address book option is choosable
ENS name option is choosable

**Comment by ethgasstation** [comment]
I agree- this should be a priority- otherwise very difficult as is to initiate send transactions from the ios app
